blinkit-app/src/homePage/molecules/loginCart/LoginCart.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import { UserConsumer } from "../../../userContext";
+import Insertbutton from "../../atoms/insertButton/Insertbutton";
+import "./loginCart.css";
+class LoginCart extends Component {
+  render() {
+    return (
+      <UserConsumer>
+        {(data) => {
+          return(
+          <div className="login-cart">
+            <div className="login">Login</div>
+            <div>
+              <Insertbutton className="cart">
+                <div className="cart-logo">🛒</div>
+                {data.total.item === 0 ? (
+                  <div id="cart-count-item">My Cart</div>
+                ) : (
+                  <div id="cart-count-item">
+                    {data.total.item} items{" "}
+                    <p>₹{data.total.price}</p>
+                  </div>
+                )}
+              </Insertbutton>
```

Comment:
InsertButton how???? please rename this as just 'button' so that it can be used at multiple places